libbeat/outputs/kafka/docs/kafka.asciidoc

```diff
@@ -201,6 +201,26 @@ NOTE: Publishing to a subset of available partitions potentially increases resou
 
 The configurable ClientID used for logging, debugging, and auditing purposes. The default is "beats".
 
+["source","yaml",subs="attributes"]
+------------------------------------------------------------------------------
+output.kafka:
+  client_id: 'beats'
+------------------------------------------------------------------------------
+
+beta[]
+
+You can set the clientID dynamically by using a format string to access any event field. For example, this configuration uses a custom field, `kubernetes.pod.annotations.tenant_name`, to set the topic for each event:
```

Comment:
It's not clear why you would want multiple clients, in this case why would the topic selector not be enough?